### ISSUE TITLE
New Maintainers

### ISSUE THREAD

**Issue by jspizziri** [oblador/react-native-progress#255]
@oblador , this library is in need of some work.

Are you interested in adding new maintainers?

**Comment by oblador** [comment]
Yep, happy to get some co-maintainers! Are you offering? Lets chat on Twitter or email if that's the case.

**Comment by jspizziri** [comment]
@oblador , well... I wish I could, but in reality, I fear I'm already maintaining too many projects 😆 . I do have a vested interest in seeing a few things get fixed, however (namely things pertaining to web support). But as I said, I don't think I could honestly offer to maintain long-term.

Perhaps there are other folks in the community who could volunteer through 😄 .

**Comment by oblador** [comment]
Nobody has offered yet unfortunately. Made some necessary updates and closing this issue for now.
